CHANGELOG.md

```diff
@@ -46,6 +46,9 @@
   visualization has been adjusted, allowing to display the values of the
   precomputed bins, which is useful when the dataset is relatively big, and it's
   cheaper to send the precomputed bins rather than the entire dataset.
+- [Hovering over the output port of a node shows the node's output type][1427].
+  This allows discovering the result type of a node which can help with
+  debugging and development.
```

Comment:
I suppose the description should be altered, as we already released the pop-ups.